drafts/25-xxx-specifications.txt

```diff
@@ -567,11 +567,121 @@ nd20. The result of evaluating a processor-dependent directive is processor-depe
+    - "left" means the operator binds to the left,
+    - "right" means the operator binds to the right.
+    - "nonassoc" means that the operator is not associative.
+
+The following table describes the semantics of the allowed operators
+in conditional expressions. The table is grouped by precedence level,
+from lowest precedence to highest.
+
+We label subexpressions "e1", "e2", and "e3" to aid in describing the
+evaluation semantics. Unless otherwise specified, all operators
+evaluate with the same semantics as their Fortran counterparts.
+
+
+| Prec |   Op    |    Syntax    |  Assoc'y |  Evaluation Semantics      |
+|------+---------+--------------+----------+----------------------------|
+| low  |   ? :   | e1 ? e2 : e3 |  right   |      conditional-expr      |
+|------+---------+--------------+----------+----------------------------|
+|      |   ¦¦    |   e1 || e2   |   left   |       Fortran .OR.         |
+|------+---------+--------------+----------+----------------------------|
+|      |   &&    |   e1 && e2   |   left   |       Fortran .AND.        |
+|------+---------+--------------+----------+----------------------------|
+|      |    ¦    |   e1 | e2    |   left   |    Fortran IOR(e1, e2)     |
+|      |    ^    |   e1 ^ e2    |   left   |    Fortran IAND(e1, e2)    |
+|      |    &    |   e1 & e2    |   left   |    Fortran IEOR(e1, e2)    |
+|------+---------+--------------+----------+----------------------------|
+|      |   ==    |   e1 == e2   |   left   | 1 if e1 == e2, 0 otherwise |
+|      |   !=    |   e1 != e2   |   left   | 1 if e1 /= e2, 0 otherwise |
+|------+---------+--------------+----------+----------------------------|
+|      |    >    |   e1 > e2    | nonassoc | 1 if e1 > e2, 0 otherwise  |
+|      |   >=    |   e1 >= e2   | nonassoc | 1 if e1 >= e2, 0 otherwise |
+|      |    <    |   e1 < e2    | nonassoc | 1 if e1 < e2, 0 otherwise  |
+|      |   <=    |   e1 <= e2   | nonassoc | 1 if e1 <= e2, 0 otherwise |
+|------+---------+--------------+----------+----------------------------|
+|      |   <<    |   e1 << e2   |   left   | Fortran ISHFT(e1, e2)      |
+|      |   >>    |   e1 >> e2   |   left   | Fortran ISHFT(e1, -e2)     |
+|------+---------+--------------+----------+----------------------------|
+|      |    +    |   e1 + e2    |   left   |             +              |
+|      |    -    |   e1 - e2    |   left   |             -              |
+|------+---------+--------------+----------+----------------------------|
+|      |    *    |   e1 * e2    |   left   |             *              |
+|      |    /    |   e1 / e2    |   left   |             /              |
+|      |    %    |   e1 % e2    |   left   |    Fortran MOD(e1, e2)     |
+|------+---------+--------------+----------+----------------------------|
+|      | unary + |     + e1     |  right   |        unary +             |
+|      | unary + |     - e2     |  right   |        unary -             |
```

Comment:
You forgot bitwise NOT and logical NOT


```suggestion
|      | unary + |     + e1     |  right   |        unary +             |
|      | unary - |     - e1     |  right   |        unary -             |
|      | unary ~ |     ~ e1     |  right   |     Fortran NOT(e1)        |
|      | unary ! |     ! e1     |  right   | 1 if e1 == 0, 0 otherwise  |
```